### PR TITLE
Add presto-doc information about new int types

### PR DESCRIPTION
**Pull request by fiedukow**

### CODE REVIEW

**Comment by fiedukow** [comment]
@raghavsethi Hey, according to @petroav you were interested in that, so - can you review this?

File: presto-docs/src/main/sphinx/language/types.rst [hunk 1]
```diff
@@ -4,6 +4,7 @@ Data Types
 
 Presto currently supports a limited set of data types.
 These types can be used with the standard ``CAST`` operator.
+Note: Not every of this type is supported by every connector, this is only list of types supported by Presto.
```
Comment (electrum) [inline presto-docs/src/main/sphinx/language/types.rst]: Use note formatting for this

File: presto-docs/src/main/sphinx/language/types.rst [hunk 2]
```diff
@@ -16,6 +17,24 @@ BIGINT
     A 64-bit signed two's complement integer with a minimum value of
     ``-2^63`` and a maximum value of ``2^63 - 1``.
 
+INTEGER
```
Comment (electrum) [inline presto-docs/src/main/sphinx/language/types.rst]: Order these from smallest to largest

**Comment by fiedukow** [comment]
@electrum comments applied

File: presto-docs/src/main/sphinx/language/types.rst [hunk 3]
```diff
@@ -5,11 +5,31 @@ Data Types
 Presto currently supports a limited set of data types.
 These types can be used with the standard ``CAST`` operator.
 
+.. note:: Not every of this type is supported by every connector, this is only list of types supported by Presto.
```
Comment (petroav) [inline presto-docs/src/main/sphinx/language/types.rst]: "Some connectors may not support all types."

This of course raises the question of which types are supported by which connectors.

Comment (fiedukow) [inline presto-docs/src/main/sphinx/language/types.rst]: Changed to what we've discussed offline.

File: presto-docs/src/main/sphinx/language/types.rst [hunk 4]
```diff
@@ -5,11 +5,31 @@ Data Types
 Presto currently supports a limited set of data types.
 These types can be used with the standard ``CAST`` operator.
 
+.. note:: Not every type has to be supported by every connector.
```
Comment (raghavsethi) [inline presto-docs/src/main/sphinx/language/types.rst]: .. note:: Connectors are not required to support all types

**Comment by raghavsethi** [comment]
Looks good to me

**Comment by fiedukow** [comment]
Comment applied.

**Comment by electrum** [comment]
Merged, thanks!

**Pull request closed without merging**